resources/views/dashboard/partials/getting-started-individual.blade.php

```diff
@@ -80,15 +80,17 @@
         <ul role="list">
             @if (Auth::user()->individual->isConsultant())
                 <li>
-                    <a href="{{ settings('ac_application') }}" rel="noopener" target="_blank">
+                    <a href="{{ settings('ac_application')[locale()] ?? settings('ac_application')['en'] }}"
```

Comment:
I think there should be a new helper function for these and the fallback locale should be based off of `config('app.fallback_locale')`. Also LSQ should probably fallback to FR first, so probably converting those interpretation locales is also needed. 

You might have a function like:

```php
if (! function_exists('settings_localized')) {
    /**
     * Retrieve a setting from the general settings table.
     *
     * @param  string|null  $key The setting key.
     * @param  string|null  $locale The requested locale for the setting to be returned in
     * @param  mixed|null  $default A default value for the setting.
     */
    function settings_localized(?string $key = null, ?string $locale = null, mixed $default = null): mixed
    {
        $locale = get_written_language_for_signed_language($locale ?? config('app.locale'));
        $settings = settings($key, []);

        return $settings[$locale] ?? $settings[config('app.fallback_locale')];
    }
}
```